README.md

```diff
@@ -9,13 +11,9 @@
 - 2016-03-31: [@seisman][seisman] 建立 GMT 问答社区
 - 2016-03-31: [@seisman][seisman] 利用腾讯企业邮箱建立域名邮箱，管理员账号为 admin@gmt-china.org
 - 2016-06-13: [@seisman][seisman] 关闭 GMT 问答社区
-
-## 文件列表
-
-- `REAMDE.md` 本文件
-- `DNS.md` 记录DNS解析的相关设置
-- `domain-receipt.png` 域名收据的截图
-
+- 2017-03-10: [@seisman][seisman] 给域名续费一年，价格 ￥135.14
+- 2018-02-18: [@seisman][seisman] 给域名续费五年，价格 ￥675.70
+- 2019-01-27: [@seisman][seisman] 将模块手册和示例整理到中文手册中，废弃 modules 和 examples 子域名
 
```

Comment:
Could we add when we created QQ groups 1 and 2?

Reply:
Done in 6662ed7ca1a78265d53d5a5748755fa29f666b10